src/interaction/externinteraction.cpp

```diff
@@ -0,0 +1,154 @@
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <openspace/interaction/externinteraction.h>
+
+#include <openspace/openspace.h>
+#include <openspace/engine/globals.h>
+#include <openspace/engine/openspaceengine.h>
+#include <openspace/engine/windowdelegate.h>
+#include <openspace/interaction/keyframenavigator.h>
+#include <openspace/interaction/navigationhandler.h>
+#include <openspace/interaction/orbitalnavigator.h>
+#include <openspace/scene/scenegraphnode.h>
+#include <openspace/scripting/scriptengine.h>
+#include <openspace/util/camera.h>
+#include <openspace/util/time.h>
+#include <openspace/util/timemanager.h>
+
+#include <ghoul/logging/logmanager.h>
+
+namespace {
+const uint32_t ProtocolVersion = 3;
+const size_t MaxLatencyDiffs = 64;
+const char* _loggerCat = "ExternInteraction";
+
+static const openspace::properties::Property::PropertyInfo BufferTimeInfo = {
+    "BufferTime",
+    "Buffer Time",
+    "" // @TODO Missing documentation
+};
+
+static const openspace::properties::Property::PropertyInfo TimeKeyFrameInfo = {
+    "TimeKeyframeInterval",
+    "Time keyframe interval",
+    "" // @TODO Missing documentation
+};
+
+static const openspace::properties::Property::PropertyInfo CameraKeyFrameInfo = {
+    "CameraKeyframeInterval",
+    "Camera Keyframe interval",
+    "" // @TODO Missing documentation
+};
+
+static const openspace::properties::Property::PropertyInfo TimeToleranceInfo = {
+    "TimeTolerance",
+    "Time tolerance",
+    "" // @TODO Missing documentation
+};
+
+} // namespace
+
+namespace openspace {
+
+ExternInteraction::ExternInteraction()
+    : properties::PropertyOwner({ "ExternInteration", "ExternInteraction" })
```

Comment:
No need to repeat the name if it is the same...  but probably better to make the second one in to "External Interaction"

Reply:
Changed